jailer/src/env.rs

```diff
@@ -112,6 +113,12 @@ impl Env {
             .parse::<u32>()
             .map_err(Error::SeccompLevel)?;
 
+        let extra_args = args
```

Comment:
I suggest changing this to `args.values_of("...").iter().flatten().map(to_string).collect()` or something like that.

Reply:
Or
```
        let extra_args = args
            .values_of("extra-args")
            .iter()
            .flat_map(to_string)
            .collect();
```

Reply:
Thanks for the suggestions, I’ve started from them and managed to find a working solution which is now in the code.